temporian/implementation/numpy/data/event.py

```diff
@@ -172,6 +172,11 @@ def from_dataframe(
             convert_date_to_duration
         )
 
+        # sort by timestamp
```

Comment:
What about adding a parameter to allow the user to disable the sorting (if the user know the data is already sorted)? Alternatively, checking that an array is sorted is cheap (compared to other operations). Maybe we could check it, and only sort if the ts are not already stored.